Sources/WebSocketKit/WebSocket.swift

```diff
@@ -135,7 +151,7 @@ public final class WebSocket {
             } else {
                 self.close(code: .protocolError, promise: nil)
             }
-        case .text, .binary:
+        case .text, .binary, .pong:
```

Comment:
Only `.pong` is checked for here but inside `.ping` is checked for.

Reply:
I'm a little unclear on what you mean here, could you please elaborate for me? I think we could have `.pong` as a case in it's own, similar to `.pong` but it just doesn't do anything but close, unless the frame hasn't finished, then it'll close on a `.protocolError`

Reply:
Ah sorry, GitHub code folding confused me. This is fine.